### ISSUE TITLE
Tilda icon - the drop down terminal app

### ISSUE THREAD

**Issue by lightonflux** [numixproject/numix-icon-theme-circle#2436]
Just installed the dropdown terminal Tilda and realised there is no numix icon for it. 

It is worth to mention that Ubuntu Mate is a popular distribution that uses ships with enabled tilda by default. 

A symlink to one of the generic terminal icons would be sufficient. 

Tilda's current logo: https://github.com/lanoxx/tilda/blob/master/tilda.png

Website: https://github.com/lanoxx/tilda

**Comment by Foggalong** [comment]
We do actually have a [Tilda icon](https://github.com/numixproject/numix-icon-theme-circle/blob/master/Numix-Circle/48x48/apps/tilda.svg). Is it not showing up on your system?

**Comment by lightonflux** [comment]
No it does not. 

Debian 8 with Gnome 3.14. It shows the old icon in the dash.

And the symlink works. 

---

So i installed the version from testing which is 1.2.4 and it works.  So i guess the problem is an hard coded icon in the earlier 1.1.12 release that is in stable. 

---


I found the [commit](https://github.com/lanoxx/tilda/commit/94ec3d851f4426bc7474063fb5ae8e8a31c37267)! 

TL;DR: tilda's icon is only themable from 1.2.4 onwards. 

Sorry for bothering you. :)

**Comment by Foggalong** [comment]
Not a problem! :D
